pkgs/by-name/sy/sysmenu/package.nix

```
@@ -0,0 +1,61 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  git,
+  gtkmm4,
+  gtk4-layer-shell,
+  wrapGAppsHook,
+  unstableGitUpdater,
+}:
+stdenv.mkDerivation {
+  pname = "sysmenu";
+  version = "unstable-13-06-2024";
+
+  src = fetchFromGitHub {
+    owner = "System64fumo";
+    repo = "sysmenu";
+    rev = "0b891e5d27f286b867d4a0984ee284a3456de851";
+    sha256 = "sha256-zI6dmS+ZqcGrG/joWouqrMNQfTRflP677kYJLJEWTBc=";
```

`hash` is preferred over sha256 for new packages
```suggestion
    hash = "sha256-zI6dmS+ZqcGrG/joWouqrMNQfTRflP677kYJLJEWTBc=";
```